sql/insert_heartbeat.sql

```diff
@@ -13,7 +13,7 @@ INSERT INTO heartbeats
     entity,
     is_write,
     language,
-    lineno,
+    CAST(lineno AS INT),
```

Comment:
The cast will have to go on the values section. On the 14th field like `CAST ($14 AS INT)`.

Reply:
That was the point, thanks!